lib/magnet/parser.rb

```diff
@@ -2,7 +2,7 @@ module Magnet
   class Parser
     TRACKS = {
       1 => /\A%(?<format>[A-Z])(?<pan>[0-9 ]{1,19})\^(?<name>[^^]*)\^\s?(?<expiration>\d{4}|\^)(?<service_code>\d{3}|\^)(?<discretionary_data>[^\?]*)\?\Z/,
```

Comment:
I don't have a test case that has a value in track 1. Should I just put it in as a similar character that was used in Track 2?